types/gov.go

```diff
@@ -12,20 +13,55 @@ const (
 	ProposalStatusInvalid = "PROPOSAL_STATUS_INVALID"
 )
 
-// GovParams contains the data of the x/gov module parameters
+type DepositParams struct {
+	MinDeposit       sdk.Coins `json:"min_deposit,omitempty" yaml:"min_deposit"`
+	MaxDepositPeriod string    `json:"max_deposit_period,omitempty" yaml:"max_deposit_period"`
+}
+
+func NewdepositParam(d govtypes.DepositParams) DepositParams {
+	return DepositParams{
+		MinDeposit:       d.MinDeposit,
+		MaxDepositPeriod: strconv.FormatInt(d.MaxDepositPeriod.Nanoseconds(), 10) + "n",
```

Comment:
I really don't think we should do this. It might cause incompatibilities very quickly. Instead we could simply use `int64` or `uint64` instead of `string` for the type of `MaxDepositPeriod`. Then we can simply do

```golang
MaxDepositPeriod: d.MaxDepositPeriod.Nanoseconds(),
```